firestore/scripts/FriendlyEats.Data.js

```diff
@@ -15,71 +15,71 @@
-  var query = firebase.firestore()
+FriendlyEats.prototype.getAllRestaurants = (render) => {
+  const query = firebase.firestore()
     .collection('restaurants')
     .orderBy('avgRating', 'desc')
     .limit(50);
   this.getDocumentsInQuery(query, render);
 };
 
-FriendlyEats.prototype.getDocumentsInQuery = function(query, render) {
-  query.onSnapshot(function (snapshot) {
+FriendlyEats.prototype.getDocumentsInQuery = (query, render) => {
+  query.onSnapshot((snapshot) => {
     if (!snapshot.size) {
       return render();
     }
 
-    snapshot.docChanges().forEach(function(change) {
+    snapshot.docChanges().forEach((change) => {
       if (change.type === 'added' || change.type === 'modified') {
         render(change.doc);
       }
     });
   });
 };
 
-FriendlyEats.prototype.getRestaurant = function(id) {
+FriendlyEats.prototype.getRestaurant = (id) => {
   return firebase.firestore().collection('restaurants').doc(id).get();
 };
 
-FriendlyEats.prototype.getFilteredRestaurants = function(filters, render) {
-  var query = firebase.firestore().collection('restaurants');
+FriendlyEats.prototype.getFilteredRestaurants = (filters, render) => {
+  const query = firebase.firestore().collection('restaurants');
 
   if (filters.category !== 'Any') {
-    query = query.where('category', '==', filters.category);
+    const query = query.where('category', '==', filters.category);
   }
 
   if (filters.city !== 'Any') {
-    query = query.where('city', '==', filters.city);
+    const query = query.where('city', '==', filters.city);
   }
 
   if (filters.price !== 'Any') {
-    query = query.where('price', '==', filters.price.length);
+    const query = query.where('price', '==', filters.price.length);
   }
 
   if (filters.sort === 'Rating') {
-    query = query.orderBy('avgRating', 'desc');
+    const query = query.orderBy('avgRating', 'desc');
   } else if (filters.sort === 'Reviews') {
-    query = query.orderBy('numRatings', 'desc');
+    const query = query.orderBy('numRatings', 'desc');
   }
 
   this.getDocumentsInQuery(query, render);
 };
 
-FriendlyEats.prototype.addRating = function(restaurantID, rating) {
-  var collection = firebase.firestore().collection('restaurants');
-  var document = collection.doc(restaurantID);
-  var newRatingDocument = document.collection('ratings').doc();
+FriendlyEats.prototype.addRating = (restaurantID, rating) => {
+  const collection = firebase.firestore().collection('restaurants');
+  const document = collection.doc(restaurantID);
+  const newRatingDocument = document.collection('ratings').doc();
 
-  return firebase.firestore().runTransaction(function(transaction) {
-    return transaction.get(document).then(function(doc) {
-      var data = doc.data();
+  return firebase.firestore().runTransaction((transaction) => {
+    return transaction.get(document).then((doc) => {
+      const data = doc.data();
 
-      var newAverage =
+      let newAverage =
```

Comment:
please change this `let` to `const` since we never change the value of `newAverage`

Reply:
Thanks for pointing that out. I over looked that. I'll need to fix it.